library/src/amd_detail/rocblaslt/src/include/rocblaslt_mat_utils.hpp

```diff
@@ -140,30 +140,31 @@ inline rocblaslt_status validateMatmulDescrArgs(rocblaslt_handle       handle,
-                                           hipblasOperation_t     opB,
-                                           int                    num_batches_a  = 1,
-                                           int                    num_batches_b  = 1,
-                                           int                    num_batches_c  = 1,
-                                           int                    num_batches_d  = 1,
-                                           int64_t                batch_stride_a = 0,
-                                           int64_t                batch_stride_b = 0,
-                                           int64_t                batch_stride_c = 0,
-                                           int64_t                batch_stride_d = 0)
+inline rocblaslt_status validateMatmulArgs(int64_t                       m,
+                                           int64_t                       n,
+                                           int64_t                       k,
+                                           const void*                   alpha,
+                                           const void*                   a,
+                                           const void*                   b,
+                                           const void*                   beta,
+                                           const void*                   c,
+                                           const void*                   d,
+                                           hipDataType                   type_a,
+                                           hipDataType                   type_b,
+                                           hipDataType                   type_c,
+                                           hipDataType                   type_d,
+                                           rocblaslt_compute_type        compute_type,
+                                           hipblasOperation_t            opA,
+                                           hipblasOperation_t            opB,
+                                           int                           num_batches_a  = 1,
+                                           int                           num_batches_b  = 1,
+                                           int                           num_batches_c  = 1,
+                                           int                           num_batches_d  = 1,
+                                           int64_t                       batch_stride_a = 0,
+                                           int64_t                       batch_stride_b = 0,
+                                           int64_t                       batch_stride_c = 0,
+                                           int64_t                       batch_stride_d = 0, 
+                                           const rocblaslt_pointer_mode& pointermode    = rocblaslt_pointer_mode_host) // add pointermode to check if alpha is in device memory.
```

Comment:
did you run clang format?
remove comment
 // add pointermode to check if alpha is in device memory.